packages/cli-repl/test/test-shell.ts

```diff
@@ -113,7 +113,23 @@ export class TestShell {
 
   async waitForPrompt(start = 0): Promise<void> {
     await eventually(() => {
-      if (!this._output.slice(start).match(PROMPT_PATTERN)) {
+      const output = this._output.slice(start);
+      const lines = output.split('\n');
+      const found = !!lines.filter(l => l.match(PROMPT_PATTERN)) // a line that is the prompt must at least match the pattern
+        .find(l => {
+          // in some situations the prompt occurs multiple times in the line (but only in tests!)
+          const prompts = l.trim().replace(/>$/g, '').split('>').map(m => m.trim().replace(/>$/g, ''));
```

Comment:
There should be no `>` characters left after the `split('>')`, so I think that last `.replace()` can be dropped?